lib/appmap/trace.rb

```diff
@@ -2,10 +2,12 @@
 
 module AppMap
   module Trace
-    class RubyMethod
+    class RubyMethod < SimpleDelegator
```

Comment:
To behave properly in a Set, inherit `hash` and `eql?` from the `method` parameter.